Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM node:13.12.0-alpine
+
+WORKDIR /app/front/
+
+#install app dependencides
+COPY package*.json ./
```

Comment:
copy `package-lock.json` as well
https://docs.npmjs.com/configuring-npm/package-lock-json.html

Reply:
I used a wild card * especially for that- see reference in [https://docs.docker.com/engine/reference/builder/#copy](url).
and also [https://www.robinwieruch.de/docker-create-react-app-development](url)